### DISCUSSION TITLE
Menu doesn't return to full available height

### DISCUSSION THREAD

**Discussion by crisshaker** [ariakit/ariakit#4130] [Questions]
https://github.com/user-attachments/assets/cdc52f01-95bf-4431-a764-7e94d9b9f7d6

In the attached, my expectation is that the menu will return to its original 'full' height after the combobox is cleared. But it doesn't. Is this expected behavior?

**Comment by benrodrs** [comment]
Could you please provide a reproduction on StackBlitz so I can take a look? Is it overlapping the trigger in the initial state?
